ethstaker_deposit/utils/constants.py

```diff
@@ -11,6 +11,7 @@
 DOMAIN_DEPOSIT = bytes.fromhex('03000000')
 DOMAIN_VOLUNTARY_EXIT = bytes.fromhex('04000000')
 DOMAIN_BLS_TO_EXECUTION_CHANGE = bytes.fromhex('0A000000')
+DOMAIN_BLS_TO_EXECUTION_CHANGE_KEYSTORE = bytes.fromhex('0B000000')
```

Comment:
This needs to be changed: https://github.com/prysmaticlabs/prysm/blob/7c213ce161cb929659c9cd96c7bfa08bcd8ef820/config/params/mainnet_config.go#L174C2-L174C21